.vscode/extensions.json

```diff
@@ -2,6 +2,6 @@
 	// See https://go.microsoft.com/fwlink/?LinkId=733558
```

Comment:
I don't know how this will affect the build now, if nothing breaks this can go in. Curious as to what does this do?